vulnerability/events_creator.go

```diff
@@ -83,7 +80,6 @@ type Scanner struct {
 }
 
 type Score struct {
-	Impact  float64 `json:"impact,omitempty"`
```

Comment:
Based on our discussion, we don't want to use a third-party library to compute a score from the vector. Therefore, there would be no benefit in sending the Impact field, which is currently being calculated inaccurately.

Reply:
Sure, I've also commented on @kfirpeled mapping PR.